pages/governance.md

```diff
@@ -0,0 +1,235 @@
+---
+title:  "Governance Structure"
+---
+
+This document describes the formal goverance structure of the JuMP project.
+
+For the purpose of this document, "JuMP" includes all repositories in the
+[JuMP-dev Github organization].
+
+## Mission
+
+The mission of the JuMP project is to provide a free, open-source, and
+high-performance software stack for mathematical programming.
+
+## Code of Conduct
+
+The JuMP community strongly values inclusivity and diversity. Everyone should
+treat others with the utmost respect. Everyone in the community must adhere to
+the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct) and the
+[Julia Community Standards](https://julialang.org/community/standards/), which
+reflect the values of our community. Both these links have information on how to
+confidentially report a violation.
+
+## Entities
+
+This section outlines the different entities that exist within the JuMP project,
+their basic role, and how membership of each entity is determined.
+
+### BDFL
+
+Due to his role in the creation of JuMP, [Miles Lubin](https://github.com/mlubin)
+holds the role of [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life).
+
+### Core Contributors
+
+Core contributors lead the technical development of the JuMP project, and they
+are the ultimate authority on the direction of the JuMP project.
+
+The current core contributors are:
+
+* [Miles Lubin (@mlubin)](https://github.com/mlubin)
+* [Benoît Legat (@blegat)](https://github.com/blegat)
+* [Joaquim Dias Garcia (@joaquimg)](https://github.com/joaquimg)
+* [Joey Huchette (@joehuchette)](https://github.com/joehuchette)
+* [Oscar Dowson (@odow)](https://github.com/odow)
+
+A new core contributor may be added by consensus of the current core
+contributors and notification to the Steering Council (see below).
+
+Before becoming a core contributor, it is expected that the community member is
+a repository maintainer (see below) of several repositories.
+
+### Emeritus Core Contributors
+
+Emeritus core contributors are community members who were core contributors, but
+are no longer active participants in the JuMP project.
```

Comment:
"... who were core contributors, but have stepped back to a less active role. A core developer may choose to switch to emeritus status by informing the other core developers and the steering council."